modules/ROOT/pages/java-platform.adoc

```diff
@@ -81,7 +81,7 @@ Console App Development::
 [source,groovy,subs=attributes+]
 ----
 dependencies {
-    implementation 'com.couchbase.lite:couchbase-lite-java:{version-full}'
```

Comment:
Is this just so that, on the web page, the version is clearly visible?  It really is best practice to use the constant, IRL.